sample_apps_reinvent2021/java/README.md

```diff
@@ -0,0 +1,38 @@
+<!-- This sample application is part of the Timestream prerelease documentation. The prerelease documentation is confidential and is provided under the terms of your nondisclosure agreement with Amazon Web Services (AWS) or other agreement governing your receipt of AWS confidential information. -->
+
+# TimestreamCustomerSampleJava
+
+Sample Java application for AWS SDK
+
+----
+## How to use it
+
+Ensure your Java SDK and runtime are 1.8 or higher.
+
+1. Install maven: https://maven.apache.org/install.html
+
+1. Go to Timestream Java sample app directory
```

Comment:
Fix numbering

Reply:
This numbering is valid and markdown will automatically increment.